src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -3614,8 +3639,8 @@ protected internal override void Dispatch()
         }
         protected internal override void Validate()
         {
-            Debug.Assert(!(Version == 0 && EventDataLength != 0 + (TypeParameterCount * 8) + 36));
-            Debug.Assert(!(Version > 0 && EventDataLength < 0 + (TypeParameterCount * 8) + 36));
+            Debug.Assert(!(Version == 0 && EventDataLength != 0 + (TypeParameterCount * 8) + 32));
+            Debug.Assert(!(Version > 0 && EventDataLength < 0 + (TypeParameterCount * 8) + 32));
```

Comment:
Is this change right? I hit an assert here in debug.

Reply:
To be honest, I'm not sure.  The size of the event is fixed, so I'm surprised that the existing code isn't right.